bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/data/ModuleType.java

```diff
@@ -199,7 +195,7 @@ public URI getConfigDescription() {
 
     public static ModuleType from(ThingTypeUID thingTypeUID) {
         String id = thingTypeUID.getId();
-        return ModuleType.AS_SET.stream().filter(mt -> mt.toUID().equals(id)).findFirst()
+        return ModuleType.AS_SET.stream().filter(mt -> mt.uid.equals(id)).findFirst()
```

Comment:
Perhaps this would be simpler?

`        return ModuleType.AS_SET.stream().filter(mt -> mt.thingTypeUID.equals(thingTypeUID)).findFirst()
`

and would remove the need for storing the variable currently named `uid`.

Reply:
Correct